CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## Master (Unreleased)
 
-## 2.21.0 (2023-XX-XX)
+## 2.21.0 (2023-05-03)
```

Comment:
Perhaps we should do a v2.22.0 instead. Then we can do a v2.21.0 release of rubocop-rspec [without the removal of factory_bot cops](https://github.com/rubocop/rubocop-rspec/blob/70a97b1895ce4b9bcd6ff336d5d343ddc6175fe6/CHANGELOG.md), followed by a v2.22.0 release with just the removal of factory_bot?

Reply:
Indeed it is! Change to v2.22.0 👍🏻